DfE.FindInformationAcademiesTrusts/Pages/OtherServicesLinkBuilder.cs

```diff
@@ -24,52 +25,47 @@ public class OtherServicesLinkBuilder : IOtherServicesLinkBuilder
     private const string FindSchoolPerformanceDataBaseUrl =
         "https://www.find-school-performance-data.service.gov.uk";
 
-    public string? GetInformationAboutSchoolsListingLink(Trust trust)
+    public string GetInformationAboutSchoolsListingLink(TrustDetailsServiceModel trust)
```

Comment:
Do we need to be passing pull objects? Not something I would ever fail a PR on but perhaps just passing trustUid might be more explicit?

Reply:
Out of scope